Source/Other/Parsing/SPSQLParser.m

```diff
@@ -1009,6 +1010,10 @@ - (NSUInteger) length {
 	return [string length];
 }
 - (unichar) characterAtIndex:(NSUInteger)anIndex {
+    if(anIndex >= string.length){
+        CLS_LOG(@"ERROR: anIndex: [%lu] >= string.len: [%lu]. string = %@", (unsigned long)anIndex, (unsigned long)string.length, safeString(string));
+        return (unichar)0x00; // FIXME: what should this return?
```

Comment:
no ieda

Reply:
I think null char is ok, we'll catch errors.